packages/alert-banner/src/alert-banner.css

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import url('./spectrum-alert-banner.css');
+
+:host {
+    --spectrum-alert-banner-size: 100%;
```

Comment:
This seems like the sort of thing we should raise as an error in the Spectrum CSS implementation.

Reply:
Got it, I removed from our styles and will raise this to `spectrum-css`